BackEnd/src/main/java/MusicAPI/harmonicsKB/Key.java

```diff
@@ -0,0 +1,11 @@
+package MusicAPI.harmonicsKB;
+
+/**
+ * Created by ben on 2/23/2016.
+ */
+public enum Key {
```

Comment:
All these objects should be in a separate folder probably to making understanding easier